.github/actions/bpftrace/scripts/check-encryption-leaks.bt

```diff
@@ -1,16 +1,34 @@
-// Six parameters expected:
+// Eight parameters expected:
 // $1: IPv4 CiliumInternalIP - Node1
 // $2: IPv6 CiliumInternalIP - Node1
 // $3: IPv4 CiliumInternalIP - Node2
 // $4: IPv6 CiliumInternalIP - Node2
```

Comment:
Citing myself:

![image](https://github.com/user-attachments/assets/b22eb8cd-4294-4779-bb8d-23a730c6d602)

I think the larger point is that, as a reviewer, it makes things easier because I only have one thing to keep in mind when reading the code, not six :grin:

Reply:
Yup, will wait for fixes to get-in, rebase, then refactor this into smaller commits. Thanks :pray: